bucket_39/libreoffice/specification

```diff
@@ -61,6 +61,7 @@ BUILD_DEPENDS=		perl-Archive-Zip:single:perl_default
 			gstreamer1:dev:standard
 			gstreamer1-plugins-base:dev:standard
 			harfbuzz:dev:standard
+			glew:primary:standard
```

Comment:
this might have been what I saw before.
on libreoffice spec, line 105 is adds "glew:primary:standard" to the buildrun section.

you are adding to the same dependency to the 
BUILD_DEPENDS section.  I think you meant to put "glew:dev:standard" here.

Reply:
The last commit ought to do it. I was just thinking this morning when I looked at the diffs on github at work and thought I mixed up something on the libreoffice specification.